pkgs/development/libraries/gsettings-desktop-schemas/default.nix

```diff
@@ -35,6 +38,10 @@ stdenv.mkDerivation rec {
     patchShebangs build-aux/meson/post-install.py
   '';
 
+  mesonFlags = [
```

Comment:
Please keep the declarative attributes before phases and hooks in GNOME packages as per https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.